development/forms.py

```diff
@@ -2,33 +2,49 @@
 import requests
 from environment import get_env_variable
 
+from auth_app.models import Bot
+
 
 SERVER_URL = get_env_variable('SERVER_URL')
 SERVER_PORT = get_env_variable('SERVER_PORT')
 
-my_bots = [
-    ('0', 'Pedro'),
-]
-
 
 class ChallengeForm(forms.Form):
     def __init__(self, *args, **kwargs):
+        user = kwargs.pop('user')
```

Comment:
Please, remove this line since you're already initiating the user in the `get_form` method.